src/autopas/containers/linkedCells/traversals/C01Traversal.h

```diff
@@ -100,8 +100,9 @@ class C01Traversal
 
   /**
    * Pairs for processBaseCell().
+   * @note std::map not applicable since ordering is important for later processing!
```

Comment:
But map is ordered D:
If you use pairs, explain what first and second is.

Reply:
> But map is ordered D:

Unfortunately, I need  the ordering arising from insertion. I'll document that.